Internal: Aligned release tools to the latest CKEditor 5 dev.

martnpaneq opened the pull request:
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Aligned release tools to the latest CKEditor 5 dev. Closes ckeditor/ckeditor5-vue#250.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

psmyrek commented on package.json:
```diff
@@ -82,8 +85,11 @@
     "dist",
-    "scripts/postinstall.js"
+    "README.md",
+    "CHANGELOG.md",
```
Previously, we did not publish the `CHANGELOG.md` - see https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code. Is it needed on npm?

martnpaneq replied:
I don't have a strong opinion about it, but we publish it for all other packages. So I guess it would be good to keep it consistent and publish it.

psmyrek reviewed (commented):
Changes work fine 🎉

Maybe we could remove the `dist` directory from the GitHub and add it to `.gitignore`?
Also, there are some remarks to `dist` directory in the `README.md` that probably should be updated.

martnpaneq commented on scripts/preparepackages.js:
```diff
@@ -0,0 +1,91 @@
+	{
+		title: 'Cleaning-up.',
+		task: () => {
+			return releaseTools.cleanUpPackages( {
+				packagesDirectory: 'release',
+				packageJsonFieldsToRemove: [
+					// TODO: Preserve `scripts.postinstall`.
+					// See: https://github.com/ckeditor/ckeditor5/issues/14318.
+					'devDependencies'
+				]
+			} );
```
We are including the postinsdtall hook in the integration packages, like:
https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code
https://www.npmjs.com/package/@ckeditor/ckeditor5-vue?activeTab=code

So I added it to keep the convention.

I guess we have to ask @pomek about it when he gets back, because I was not able to figure out if it's required or not.

martnpaneq commented:
> Changes work fine 🎉
> 
> Maybe we could remove the `dist` directory from the GitHub and add it to `.gitignore`? Also, there are some remarks to `dist` directory in the `README.md` that probably should be updated.

Removed `dist` from git tracking, and updated `README.md`.

psmyrek reviewed (approved):
LGTM 🎉

pomek reviewed (approved):
LGTM.
